config.yml

```diff
@@ -2757,18 +2757,56 @@ nodes:
     fields:
       - name: keyword_loc
         type: location
+        comment: |
+          The location of the `unless` keyword.
+
+              unless cond then bar end
+              ^^^^^^
+
+              bar unless cond
+                  ^^^^^^
       - name: predicate
         type: node
+        comment: |
+          The condition to be evaluated for the unless expression. Can be any
+          kind of node that represents a non-void expression.
```

Comment:
I couldn't find where/how we define "non-void expressions". Documenting that (if it's not present) would be useful too.